### PR TITLE
Change dates

### PR DESCRIPTION
**Pull request by enric4000**
## Which issue(s) this PR fixes (optional) 

Fixes #
References #

## Some questions
- [ ] I have read the contributing guidelines
- [ ] I abide by this repository Code of Conduct
- [ ] I understand that my PR won't be merged until GitHub gives a "green light"

### CODE REVIEW

**Review by EncryptEx — DISMISSED** [review]
LGTM

File: app/static/css/bootstrap/mixins/_banner.scss [hunk 1]
```diff
@@ -1,8 +1,8 @@
 @mixin bsBanner($file, $suffix:"") {
   /*!
    * Bootstrap #{$file} v5.2.0 (https://getbootstrap.com/)
-   * Copyright 2011-2022 The Bootstrap Authors
-   * Copyright 2011-2022 Twitter, Inc.
+   * Copyright 2011-2023 The Bootstrap Authors
+   * Copyright 2011- Twitter, Inc.
```
Comment (carlotacb) [inline app/static/css/bootstrap/mixins/_banner.scss:5]: Add 2023

File: application/forms/base.py [hunk 2]
```diff
@@ -11,12 +11,12 @@
...
 
 
-YEARS = [(year, str(year)) for year in range(timezone.now().year - 1, timezone.now().year + 6)]
+YEARS = [(year, str(year)) for year in range(timezone.now().year, timezone.now().year + 7)]
 DEFAULT_YEAR = timezone.now().year + 1
 EXTENSIONS = getattr(settings, 'SUPPORTED_RESUME_EXTENSIONS', None)
 
 HACK_NAME = getattr(settings, 'HACKATHON_NAME')
-EXTRA_NAME = [' 2016 Fall', ' 2016 Winter', ' 2017 Fall', '  2017 Winter', ' 2018', ' 2019', ' 2021', ' 2022']
+EXTRA_NAME = [' 2019', ' 2021', ' 2022']
```
Comment (carlotacb) [inline application/forms/base.py:19]: Add 2020

File: application/forms/base.py [hunk 3]
```diff
@@ -11,12 +11,12 @@
 from application.models import Application
 
 
-YEARS = [(year, str(year)) for year in range(timezone.now().year - 1, timezone.now().year + 6)]
+YEARS = [(year, str(year)) for year in range(timezone.now().year, timezone.now().year + 7)]
```
Comment (carlotacb) [inline application/forms/base.py:14]: Why did you remove -1

File: user/forms.py [hunk 4]
```diff
@@ -78,7 +78,7 @@ def clean_password2(self):
         return password2
 
     def save(self, commit=True):
-        # Save the provided password in hashed format
+        # Save the provided password in hashed
```
Comment (carlotacb) [inline user/forms.py]:
```suggestion
        # Save the provided password in hashed format
```

File: application/forms/base.py (hunk 2 above)
Comment (carlotacb) [inline application/forms/base.py:19]:
```suggestion
EXTRA_NAME = [' 2019', '2020', 2021', ' 2022']
```

**Review by carlotacb — DISMISSED** [review]

**Review by carlotacb — DISMISSED** [review]

**Review by carlotacb — APPROVED** [review]

**Review by carlotacb — APPROVED** [review]
